java/jtqj/core/src/main/java/com/devonfw/application/jtqj/general/common/impl/security/CsrfRequestMatcher.java

```diff
@@ -12,12 +13,13 @@
  *
  * @see <a href="https://en.wikipedia.org/wiki/Cross-site_request_forgery">Cross-site request forgery</a>
  */
+@Named("CsrfRequestMatcher")
 public class CsrfRequestMatcher implements RequestMatcher {
```

Comment:
Do we need this class in latest version of devon4j?

Reply:
yes.. it is required.